http/server.ts

```diff
@@ -570,6 +593,58 @@ export async function serve(
   return await server.serve(listener);
 }
 
+/** Serve HTTP requests with the given handler
+ *
+ * ```ts
+ * import { serve } from "https://deno.land/std@$STD_VERSION/http/server.ts";
+ * serve((_req) => new Response("Hello, world"));
+ * ```
+ *
+ * ```ts
+ * import { serve } from "https://deno.land/std@$STD_VERSION/http/server.ts";
+ * console.log("server is starting at localhost:3000");
+ * serve((_req) => new Response("Hello, world"), { addr: ":3000" });
+ * ```
+ *
+ * ```ts
+ * import { serve } from "https://deno.land/std@$STD_VERSION/http/server.ts";
+ * const certFile = "/path/to/certFile.crt";
+ * const keyFile = "/path/to/keyFile.key";
+ * console.log("server is starting at https://localhost:8000");
+ * serve((_req) => new Response("Hello, world"), {
+ *   tls: { certFile, keyFile },
+ * });
+ * ```
+ */
```

Comment:
If only for consistency (but also richer documentation) it would be good to:
1. Include the description/defaults/caveats similar to the other methods, including details on default host and ports (which here are 0.0.0.0 and 8000 opposed to the other methods)
2. Add descriptions to each code block to point out what they show
3. Add the jsdoc `@param`

Reply:
Added docs!